wheelctrl_ros2:  
  ros__parameters:
    robot_param:
      name: "Elephant Robot"
    
    moving_wheel:
      type_name: "omni"
      radius: 0.051
      quantity: 4
      loop_rate: 200.0
      gear_ratio: 1.0
      coordinate: "polar"
      distance: 0.7
      arguments: [45.0,135.0,225.0,315.0]
      hardID: [0,1,2,3]
    
    measuring_wheel:
      type_name: "omni"
      radius: 0.03
      quantity: 3
      loop_rate: 200.0
      gear_ratio: 1.0
      coordinate: "polar"
      distance: 0.1
      arguments: [0.0,120.0,240.0]
      hardID: [0,1,2]